components/base/Collapsible.tsx

```diff
@@ -78,15 +85,33 @@ function Collapsible({ triggerText, children }: Props) {
 
   const transitionTime = 200;
 
+  // Sets active state on collapsible
+  function handleActive() {
+    if (!active) {
+      setActive(true);
+    } else {
+      setActive(false);
+    }
+  }
+
+  // Handle "Enter" key to open/close collapsible
+  function handleKeyDown(e) {
```

Comment:
Pressing enter when the the collapsible is **open**, will **close** it, and vice versa.